### PR TITLE
COG-989 MFA timer seconds should be in two digits when it is below 10 seconds (0:9 vs 0:09)

### PR DESCRIPTION
**Pull request by Nasir2712**

### CODE REVIEW

File: src/utils/CommonHelper.js [hunk 1]
```diff
@@ -9,3 +9,12 @@ export const customErrorMessage = (errorMessage) => {
   }
   return customMessage[errorMessage] || customMessage.default
 }
+
+export const countDownConverter = countDown => {
```
Comment (kelfink) [inline src/utils/CommonHelper.js]: I recommend a rename of this method.  It's not really to do with countdowns, but with time, right?  So maybe name it the most obvious, after its function:  secondsToTime(seconds).

Comment (kelfink) [reply src/utils/CommonHelper.js]: That will make it more reusable as well.

Comment (Nasir2712) [reply src/utils/CommonHelper.js]: ok

File: src/utils/CommonHelper.js [hunk 2]
```diff
@@ -9,3 +9,12 @@ export const customErrorMessage = (errorMessage) => {
...
+  const secondsPerMinute = 60
+  const tenSeconds = 10
+  const min = Math.floor(countDown / secondsPerMinute)
+  const sec = countDown % secondsPerMinute
+  const seconds = sec < tenSeconds ? `0${sec}` : sec
+  return `${min}:${seconds}`
+}
```
Comment (kelfink) [inline src/utils/CommonHelper.js]: given the few cases for this padding, I think this is ok.  I would probably have used the literals 10 and 60 for a case like this, especially when we know how minutes and seconds work (evoking common knowledge). But it's fine.  For a more complex scenario, looks like we're already dependent on the 'https://www.npmjs.com/package/left-pad' library and so could just use that.

Comment (Nasir2712) [reply src/utils/CommonHelper.js]: sure

**Pull request merged**